Documentation/k8s-monitoring.md

```diff
@@ -64,3 +64,13 @@ kubectl -n rook delete statefulset prometheus-rook-prometheus
 kubectl delete -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.8/bundle.yaml
 ```
 Then the rest of the instructions in the [Prometheus Operator docs](https://github.com/coreos/prometheus-operator#removal) can be followed to finish cleaning up.
+
+## Special Cases
+
+### Tectonic Bare Metal
+In order to be able to monitor the Rook cluster on a Tectonic Bare Metal installation, you will need to create a ServiceMonitor.
+This ServiceMonitor can be tracked by a Prometheus instance which can be created using the [Prometheus Operator](https://coreos.com/operators/prometheus/docs/latest/). Tectonic strongly discourages the `tectonic-system` Prometheus instance to be used outside their intentions, so you need to create a new Prometheus instance yourself.
+
+```bash
+kubectl create -f tectonic-service-monitor.yaml
```

Comment:
This can be removed as the existing ServiceMonitor manifest already covers it.

Reply:
Can you please explain a little more. I'm not sure what you mean... 
We're talking about line 72? I don't see the ServiceMonitor covering the Operator itself.
But I guess I misunderstood :-)

Reply:
@hacor I mean that this special case to use `kubectl create -f tectonic-service-monitor.yaml` isn't needed as it is already explained above, when the `tectonic-service-monitor.yaml` file is doing the same as the existing ServiceMonitor (https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/monitoring/service-monitor.yaml).

Edit: See comments below.